prow/config.yaml

```diff
@@ -167,7 +167,7 @@ presubmits:
         - "--clean"
         - "--git-cache=/root/.cache/git"
         - "--"
-        - "--node-args=--image-config-file=test/e2e_node/image-config.yaml"
+        - "--node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml,--test-suite=cadvisor"
```

Comment:
it probably should be `/go/src/k8s.io/test-infra`, or you can keep the image config relative path and set `--image-config-dir` instead

Reply:
aka https://github.com/dashpole/test-infra/blob/0173aa99c56abaf54cbfd065b5234ec83a6a8174/prow/config.yaml#L717

Reply:
That is what we use on all of the node-e2e tests.  Why would it change for these tests?

Reply:
The containerd test looks like it is using the image config from k/k, right?  I want to use the one in test-infra

Reply:
oh right, I read it wrong